### PR TITLE
Bug 1466604 - rewrite everything to be at /docs

### PR DESCRIPTION
**Pull request by djmitche**
Landing this will need to be coordinated with adding redirects via a Lambda function in the cloudformation distribution.

This is basically the only way I found to get to the point where we can build both a redeployable and non-redeployable docs.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -168,17 +189,17 @@ function presentations() {
 // a stream creating the entire site
 function site() {
   return merge(
-    index(),
-    resources(),
-    people(),
-    error(),
-    tutorial(),
-    manual(),
+    //index(),
```
Comment (imbstack) [inline gulpfile.js]: Did you mean to uncomment these?

File: gulpfile.js [hunk 2]
```diff
@@ -111,6 +121,17 @@ function manual() {
   )
 }
 
+show = function(context) {
```
Comment (imbstack) [inline gulpfile.js]: Leftover debug?

**Comment by djmitche** [comment]
I *think* that this will not delete the files at /.  But just in case, I have the lambda function ready to go.

**Comment by djmitche** [comment]
I was wrong.  It deleted everything.  Weird, since we don't use gulp-awspublish.publisher.sync.

**Pull request merged**